docs/moduleAna4Stats.rst

```diff
@@ -40,11 +40,21 @@ In order to run this example:
       python3 runScripts/runProbAna.py
 
 
-.. figure:: _static/avaHockeyChannel_probMap.png
+.. figure:: _static/avaHockeyChannel_probMap_lim1.0.png
     :width: 90%
 
       Probability map example.
 
+Another example is given in :py:mod:`runScripts.runProbAnaCom1DFA.py`, but here only
+one parameter is varied at a time.
```

Comment:
```suggestion
An example utilising com1DFA is given in :py:mod:`runScripts.runProbAnaCom1DFA.py`, with only one parameter being varied at a time.
```

Reply:
In both examples com1DFA is used to create the simulations and then create the probability maps using these simulations, the point I wanted to make here is that with this script - it is possible to vary one parameter at a time, and then set probability configurations for creation of the maps, several possible- but it is a good point that for both example scripts com1DFA is used